ote_sdk/ote_sdk/usecases/exportable_code/demo/demo_package/utils.py

```diff
@@ -0,0 +1,86 @@
+"""
+Utils for demo
+"""
+# INTEL CONFIDENTIAL
+#
+# Copyright (C) 2021 Intel Corporation
+#
+# This software and the related documents are Intel copyrighted materials, and
+# your use of them is governed by the express license under which they were provided to
+# you ("License"). Unless the License provides otherwise, you may not use, modify, copy,
+# publish, distribute, disclose or transmit this software or the related documents
+# without Intel's prior written permission.
+#
+# This software and the related documents are provided as is,
+# with no express or implied warranties, other than those that are expressly stated
+# in the License.
+
+import importlib
+import json
+import os
+from pathlib import Path
+
+from openvino.model_zoo.model_api import models
+from openvino.model_zoo.model_api.adapters import OpenvinoAdapter, create_core
+
+from ote_sdk.entities.label import Domain
+from ote_sdk.usecases.exportable_code.prediction_to_annotation_converter import (
+    create_converter,
+)
+
+
+def get_model_path(path):
+    """
+    Get path to model
+    """
+    model_path = path
+    if model_path is None:
+        model_path = Path(__file__).parent / "model.xml"
+        if not os.path.exists(model_path):
+            raise IOError("The path to the model was not found.")
+
+    return model_path
+
+
+def get_parameters(path):
+    """
+    Get hyper parameters to creating model
+    """
+    parameters_path = path
+    if parameters_path is None:
+        parameters_path = Path(__file__).parent / "config.json"
+        if not os.path.exists(parameters_path):
```

Comment:
Here we are using both `Path` and `os.path` packages. Would be good to use one of them.

Reply:
Something like,

```
parameters_path = Path(__file__).parent / "config.json"
        if not parameters_path.exists():
```